### PR TITLE
Goshko Volodymyr hw7

### PR DESCRIPTION
**Pull request by Ant3000**

### CODE REVIEW

File: GOshko/Ghw7/GoshkoHW7Task02.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+sort_list = sorted(alike)
+alike = ''
+for i in sort_list:
+    alike += i
+sort_list = sorted(divers)
+divers = ''
+for i in sort_list:
+    divers += i
+print('the same elements :', alike)
+print('separately elements :', divers)
```
Comment (katemalash) [inline GOshko/Ghw7/GoshkoHW7Task02.py:21]: Good Decision

File: GOshko/Ghw7/GoshkoHW7Task01.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+inp_str = input('enter string : ')
+n = len(inp_str )
+res_str = inp_str[0]
+for i in range(1, n):
+    if inp_str[i-1] != inp_str[i]:
+        res_str += inp_str[i]
+print(res_str)
```
Comment (katemalash) [inline GOshko/Ghw7/GoshkoHW7Task01.py:7]: Good Decision

**Review by katemalash — APPROVED** [review]
Home Work 7 is done good. You can merge pull-request.

File: GOshko/Ghw7/GoshkoHW7Task03.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+UKR = ['Kiev', 'Lviv', 'Odessa']
+PLN = ['Warszawa', 'Krakow', 'Gdansk']
+DE = ['Berlin', 'Frankfurt', 'Hamburg']
+FRA = ['Paris', 'Marseille', 'Lyon']
+UK = ['London', 'Liverpool', 'Manchester']
+ITA = ['Roma', 'Firenze', 'Milano']
+location = {'Ukraien': UKR, 'Polska': PLN, 'Deutschland': DE, 'France': FRA, 'United Kingdom': UK, 'Italia': ITA}
```
[the remark this answers is not in the document]
Comment (Ant3000) [reply GOshko/Ghw7/GoshkoHW7Task03.py:7]: ups...

**Pull request merged**